en/docs/develop/extending-api-manager/extending-key-management/extending-the-key-manager-interface.md

```diff
@@ -12,21 +12,15 @@ For instance,
 
 -   At this point, the Developer Portal talks to the **KeyManager** to create an *OAuth client* and get the *Consumer Key/Secret* and the *Application Access token*.
 
--   When the Gateway receives a request, it talks to **KeyManager** and get the token validated. The **KeyManager** checks if the token is active, and whether the token is usable to invoke the resource being accessed. If the token is valid, the **KeyManager** sends additional details about the token (i.e., the Throttling Tier for the subscription and Consumer key) to the Gateway in the response. In turn the Gateway uses these details to determine if the request should be passed to the backend or not.
+-   When the Gateway receives a request, it talks to **KeyManager** and get the token validated. The **KeyManager** checks if the token is active, and whether the token is usable to invoke the resource being accessed. If the token is valid, the Gateway uses additional details about the token (i.e., the Throttling Tier for the subscription and Consumer key) to determine if the request should be passed to the backend or not.
 
 Therefore, the **KeyManager** interface acts as the bridge between the OAuth Provider and WSO2 API Manager (WSO2 API-M).
 
 ### Extending the Key Manager Interface
 
-When you need to write your own implementation to plug an external OAuth2 authorization server which will act as the Key Manager, you should implement the [**KeyManager** interface](https://github.com/wso2/carbon-apimgt/blob/master/components/apimgt/org.wso2.carbon.apimgt.api/src/main/java/org/wso2/carbon/apimgt/api/model/KeyManager.java) which is a Java extension point in WSO2 API-M.
+When you need to write your own implementation to plug an external OAuth2 authorization server which will act as the Key Manager, you should implement a third party Key Manager Connector as explain in 
+[Configure a Custom Key Manger]({base_path}/administer/key-managers/configure-custom-connector/)
```

Comment:
Link is defined wrong as a set of {} is missing

"{base_path}/administer/key-managers/configure-custom-connector/" should be
"{{base_path}}/administer/key-managers/configure-custom-connector"